Lib/test/test_sys.py

```diff
@@ -2114,11 +2114,18 @@ def audit_hook(event, arg):
         script = '''
 print("Remote script executed successfully!")
 '''
+        remote_exec_event_triggered = False
+        def audit_hook(event, arg):
+            if event == "remote_exec":
+                nonlocal remote_exec_event_triggered
+                remote_exec_event_triggered = True
+        sys.addaudithook(audit_hook)
```

Comment:
It's not possible to clear audit hooks, so move this test to test_audit which is designed for that: see `Lib/test/audit-tests.py`.